pkg/daemon/handler.go

```diff
@@ -201,8 +203,8 @@ func (csh cniServerHandler) handleDel(req *restful.Request, resp *restful.Respon
 	}
 
 	klog.Infof("delete port request %v", podRequest)
-	if podRequest.Provider == util.OvnProvider {
-		err = csh.deleteNic(podRequest.PodName, podRequest.PodNamespace, podRequest.ContainerID, podRequest.DeviceID)
+	if podRequest.Provider == util.OvnProvider || podRequest.CniType == util.CniTypeName {
```

Comment:
check `CniType` should be enough here

Reply:
By default, there is no Multus-CNI and CniType is empty, so I think the OVNProvider still needs to be checked here